src/utils.js

```diff
@@ -34,3 +34,33 @@ module.exports.reflect = async function (thenable, args) {
     return error;
   }
 };
+
+/**
+ * Creates promise that will resolve after the given time (in ms).
+ */
+
+module.exports.timeout = function (ms, returnValue) {
+  return new Promise(resolve => setTimeout(() => resolve(returnValue), ms));
+};
+
+/**
+ * Validate logger.
+ */
+
+module.exports.validateLogger = function (logger) {
+  if (typeof logger !== 'object') {
+    throw new Error(`Logger instance is invalid`);
+  }
+  const requiredLogMethods = ['info', 'warn', 'error'];
```

Comment:
```suggestion

  const requiredLogMethods = ['info', 'warn', 'error'];
```

Though I don't think it is the best place to put this.
This feels like a constant in a module dedicated to the logger, but provided the way it's already implemented, we can go with it.

Reply:
I don't think we have to create a new file to hold an array, but if we ever change to ts, then we can create a proper interface and place this there.

Reply:
Still needs a `\n` above it though.